server/cp/ingest/parser/cp_onclusive.py

```diff
@@ -21,7 +21,9 @@ class CPOnclusiveFeedParser(OnclusiveFeedParser):
     def parse(self, content, provider=None):
         onclusive_cv_items = _get_cv_items("onclusive_ingest_categories")
         anpa_categories = _get_cv_items("categories")
-        event_types_cvs = _get_cv_items("CP_event_types")
+        event_types = _get_cv_items("event_types")
+        cp_event_types = _get_cv_items("cp_event_types")
```

Comment:
imo these should be rather `onclusive_event_types` because `event_types` are actually the ones used by cp